_posts/2024-04-15-heapdump-oome-native.adoc

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: 'Obtaining heap dump on OutOfMemoryError with Quarkus native'
+date: 2024-04-15
+tags: native heapdump oome epsilongc jol
```

Comment:
I don't think it's a good idea to add tags that are so precise. We already have too many tags. Let's only have `native`?